storage/tokudb/ha_tokudb_admin.cc

```diff
@@ -546,19 +543,69 @@ int standard_t::analyze_key_progress(void) {
         if ((_thd && thd_killed(_thd)) || cancelled()) {
             // client killed
             return ER_ABORTING_CONNECTION;
-        } else if(_time_limit > 0 &&
-                  (uint64_t)_key_elapsed_time > _time_limit) {
+        } else if (_time_limit > 0 &&
+                   (uint64_t)_key_elapsed_time > _time_limit) {
```

Comment:
Hmm, yeah, good catch, the code existed before (where I should have used static_cast then as well) and was just relocated for this issue, but relocation/touching is grounds for correcting anything to current standards.